src/components/LabelledCard.vue

```diff
@@ -1,16 +1,15 @@
 <template>
   <div
-    class="mt-4 border-t-4 border-sp rounded-md space-y-1 bg-sdb-400 text-gray-100"
+    class="border-2 border-gray-200 rounded-md bg-white text-sp-400 shadow-[#F06292_8px_8px_0_0]"
```

Comment:
Would it be worth adding this as a boxShadow in tailwind.config
  boxShadow: {
        'custom-pink': '8px 8px 0 0 #F06292',
      },

Reply:
Yes, good idea.